woocommerce/pinterest-for-woocommerce
# Fix/228 wrong feed registration status multiple feed profiles

### Changes proposed in this Pull Request:



Closes #228 .

When a merchant has multiple feeds, we should get the feed report, in order to populate the status of the feed, using the stored feed_id. 

### Screenshots:
![image](https://user-images.githubusercontent.com/4016167/145265426-327a9c4a-38f8-4ee3-8c40-14fefa9c4d72.png)




### Detailed test instructions:

1. Create a feed profile using https://www.pinterest.com/product-catalogs/ directly and then disable it.
2. Install & finish onboarding of the plugin so that a new feed profile is registered by the plugin.
3. Verify that the status shown in the product catalog page is of the expected feed (the one created by the plugin).


### Changelog entry

> Fix - Get feed report for the stored Feed ID instead for the first feed returned by the API.
> Add - get_merchant_feed_report() method that fetches the feed_report
> Tweak - get_merchant_feed() now fetches the feed instead of the feed_report.
